### PR TITLE
logging: add exportlogs sample

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

**Comment by rakyll** [comment]
This is hard to test against the prod service given you need an actual log to be exported. Other languages are using mocks or not testing these calls at all.

/cc @broady @jba

File: logging/exportlogs/exportlogs.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	it := client.Sinks(ctx)
+	for {
+		sink, err := it.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			return nil, err
+		}
+		sinks = append(sinks, sink)
```
Comment (broady) [inline logging/exportlogs/exportlogs.go]: `sinks = append(sinks, sink.ID)` perhaps?

return a `[]string`

Demonstrates a small amount of manipulation

File: logging/exportlogs/exportlogs.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample exportlogs lists, creates, updates, and deletes log sinks.
+package main
+
+import (
+	"context"
```
Comment (broady) [inline logging/exportlogs/exportlogs.go]: wrong context

**Comment by jba** [comment]
There is a fake in logging/internal/testing. I could pull it out if it's useful. But a mock is OK by me.

**Comment by broady** [comment]
Probably not worth it for testing this.

The only thing worth testing here is the format of the sink destination (`storage...../bucket`)

**Comment by rakyll** [comment]
Done.

> There is a fake in logging/internal/testing

Let's wait until we decide whether to move these snippets to the actual repo. Then, we can share testing libs without having to export the internal stuff.

**Pull request merged**